02_analysis_scripts/README.md

```diff
@@ -0,0 +1,32 @@
+## Analysis
+Below are exmaples of how to use analysis scripts to analyse the dataset for F@H data in [DOI link]. It is recommened to run each of these scripts in separate directories.
+
+**Note**: these scripts were developed as part of a time sensive project, as a result they are not memory efficient.
+
+### Contact analysis
+`analyze_antibody_rbd_contacts.py` can be used to generate a series of `.pkl` files containing an N x M x No. frames array of distances between heavy atoms at the RBD:Ab interface.
+
+For example, analysing the F@H `17341` project (S2H97:Ab):
```

Comment:
```suggestion
For example, analysing the F@H `17347` project (RBD:S309):
```

Reply:
Can you add a line above this that says something like: 
Project numbers:
RBD:S309 - 17341
RBD:S2H97 - 17347